pkgs/development/haskell-modules/generic-builder.nix

```diff
@@ -33,9 +33,9 @@ in
 , profilingDetail ? "exported-functions"
 # TODO enable shared libs for cross-compiling
 , enableSharedExecutables ? false
-, enableSharedLibraries ? (ghc.enableShared or false)
+, enableSharedLibraries ? !stdenv.hostPlatform.isStatic && (ghc.enableShared or false)
```

Comment:
It might be nice to have a quick note here explaining that `hostPlatform.isStatic` means that we don't want any shared libraries built (or maybe there are not available to link against?).

(Actually I don't know this, I'm just guessing from the names... which is why it would be nice to have a comment about it.)

Reply:
@cdepillabout I thought a bit about out; since this is always what `isStatic` is for I think it would be better to add more documention where it is defined instead, rather than at every use site.